config/index.js

```diff
@@ -10,24 +10,27 @@ var SCOPES = ['https://www.googleapis.com/auth/firebase.remoteconfig'];
 
 /**
  * Get a valid access token.
+ *
+ * This method assumes it is either running in a trusted Google environment like GCP or if running
```

Comment:
Suggest disambiguating "it" with "your server" or whatever fits.

Reply:
@egilmorez not sure what you mean here could you give more detail?

Reply:
@egilmorez updated the docs here, could you have another look?